config/params.php

```diff
@@ -1,20 +0,0 @@
-<?php
-
-declare(strict_types=1);
-return [
```

Comment:
Why removing it?

Reply:
Because it is not necessary and these parameters are not correct at all.

Reply:
OK...